templater/jobs/presubmit/eks-anywhere-build-tooling/source-controller-presubmits-arm64.yaml

```diff
@@ -0,0 +1,23 @@
+jobName: source-controller-tooling-presubmit-arm64
+runIfChanged: EKS_DISTRO_MINIMAL_BASE_GIT_TAG_FILE|^build/lib/.*|Common.mk|projects/fluxcd/source-controller/.*
+cluster: prow-postsubmits-cluster
+serviceaccountName: postsubmits-build-account
+bucket: s3://prowdataclusterstack-316434458-prowbucket7c73355c-1n9f9v93wpjcm
```

Comment:
I thought these were set by the templater?

I think we are also missing the arch tag here